shared/editor/embeds/Tldraw.tsx

```diff
@@ -7,13 +7,13 @@ function Tldraw(props: Props) {
     <Frame
       {...props}
       src={props.attrs.href}
-      title="Tldraw.com Embed"
+      title="Tldraw.com (beta) Embed"
```

Comment:
This "beta" here isn't necessary?

Reply:
I have put bêta here and everywhere else to make sure people aren't confused. Indeed, at this moment of time, tldraw.com is still up and running. So if people paste a multiplayer link from tldraw.com, it won't work and they won't understand why. Only a multiplayer link from beta.tldraw.com will work.